### PR TITLE
Swagger changes to introduce REST APIs for offline and online region for the Cosmos DB account

### PR DESCRIPTION
**Pull request by arsalana-microsoft**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/cosmos-db.json [hunk 1]
```diff
@@ -337,6 +337,76 @@
...
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The offline region operation is completed successfully."
+          },
+          "202": {
+            "description": "Accepted. The offline region operation will complete asynchronously."
+          }
+        }
```
Comment (anuchandy) [inline specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/cosmos-db.json]: seems this is a long running operation, if so please add  `"x-ms-long-running-operation": true` attribute for both operations

**Review by anuchandy — CHANGES_REQUESTED** [review]
See review comment on LRO.

File: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/cosmos-db.json (hunk 1 above)
Comment (arsalana-microsoft) [reply specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/cosmos-db.json]: fixed.

**Review by sujitvk1 — APPROVED** [review]

**Comment by anuchandy** [comment]
@ravbhatnagar could you please take a look?

**Comment by anuchandy** [comment]
@ravbhatnagar can you please take a look?

File: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/cosmos-db.json [hunk 2]
```diff
@@ -337,6 +337,88 @@
...
+            "name": "regionParameterForOffline",
+            "in": "body",
+            "required": true,
+            "description": "Cosmos DB region to offline for the database account.",
+            "schema": {
+              "$ref": "#/definitions/RegionForOnlineOffline"
+            }
+          }
+        ],
+        "responses": {
```
Comment (ravbhatnagar) [inline specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/cosmos-db.json]: Add a "default" response to capture all the error responses. The schema complies with ARM RPC error contract. Example can be found Batch RP swagger spec.

**Comment by ravbhatnagar** [comment]
Just one minor thing. Rest looks good. Signing off from ARM.

**Review by anuchandy — APPROVED** [review]

**Pull request merged**